sacred/messagequeue.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# coding=utf-8
+import sys
+
+if sys.version_info[0] == 2:
+    from Queue import Queue
+else:
+    from queue import Queue, Empty
+
+
+class SacredMQ:
```

Comment:
Please use `class SacredMQ(object):` for new-style clases since Sacred tries to maintain python 2 compatibility